### PR TITLE
Fix typos and remove latex commands

### PR DESCRIPTION
**Pull request by h0lyalg0rithm**

### CODE REVIEW

**Comment by h0lyalg0rithm** [comment]
@nderjung

File: content/en/docs/concepts/design-principles.md [hunk 1]
```diff
@@ -43,69 +43,67 @@ unikernel project, or from scratch.
...
-  and so-called \emph{factions} that can be run on their own (e.g.  network
+  and so-called factions that can be run on their own (e.g.  network
   or filesystem support). Rump goes some way towards achieving our goals,
   however there are still many dependencies left which require that all kernels
   have the base and hypercall layers.
-  %; additionally, the dependencies on the host
-  %are limiting in the context of a VM, which is our target deployment.
+  Additionally, the dependencies on the host
+  are limiting in the context of a VM, which is our target deployment.
 }
```
Comment (StefanJum) [inline content/en/docs/concepts/design-principles.md:85]:
```suggestion
```

**Review by StefanJum — COMMENTED** [review]
Looks good besides the one suggestion, thank you.

**Comment by StefanJum** [comment]
Please squash your commits and change the commit message to match the [required format](https://unikraft.org/docs/contributing/coding-style/#commit-message-format).
It should look somewhat like `concepts/design-principles: short description of the changes`.

**Comment by gabrielmocanu** [comment]
@StefanJum  Is this good to go now?

**Review by StefanJum — APPROVED** [review]
All good now, thanks.
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Comment by StefanJum** [comment]
> @StefanJum Is this good to go now?

Thanks for the reminder @gabrielmocanu!

**Pull request merged**